mlflow/projects/backend/local.py

```diff
@@ -88,8 +97,25 @@ def run(
         elif env_manager == _EnvManager.CONDA:
             tracking.MlflowClient().set_tag(active_run.info.run_id, MLFLOW_PROJECT_ENV, "conda")
             command_separator = " && "
-            conda_env_name = get_or_create_conda_env(project.conda_env_path)
+            conda_env_name = get_or_create_conda_env(project.env_config_path)
             command_args += get_conda_command(conda_env_name)
+        elif env_manager == _EnvManager.VIRTUALENV:
+            tracking.MlflowClient().set_tag(
+                active_run.info.run_id, MLFLOW_PROJECT_ENV, "virtualenv"
+            )
+            command_separator = " && "
+            if project.env_type == env_type.CONDA:
+                python_env = _PythonEnv.from_conda_yaml(project.env_config_path)
+            else:
+                python_env = _PythonEnv.from_yaml(project.env_config_path)
```

Comment:
@dbczumar Added logic to handle a conda project.

Reply:
@harupy Tested this out, and it generally seems to work! One small issue:

If there are non-pip dependencies listed in `conda.yaml`, we don't warn the user that these won't be installed.

Reply:
Ok, can we insert a logger in `from_conda_yaml` and make it print the warning.

Reply:
Added!